ArmVirtPkg/ArmVirt.dsc.inc

```diff
@@ -381,8 +381,7 @@
 
   #
   # Enable NX memory protection for all non-code regions, including OEM and OS
-  # reserved ones, with the exception of LoaderData regions, of which OS loaders
-  # (i.e., GRUB) may assume that its contents are executable.
+  # reserved ones.

```

Comment:
Thank you for this patch.

Would it make sense to also include some text to highlight the fact that the PCD can be overridden from the build command line, should one wish to use an older GRUB build, please?

> E.g,, append this to the build.sh command line
> --pcd PcdDxeNxMemoryProtectionPolicy=0xC000000000007FD1

Ref: https://edk2.groups.io/g/devel/message/97865